### PR TITLE
PrasadC_Assignment6

### PR DESCRIPTION
**Pull request by PrasadCOct2020**
Please review my PR.

Thanks!

### CODE REVIEW

File: src/prasad/Shop.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	int oilPouch = 0;
+	int paniPuri = 43;
+	int masalaPackets = 73;
+
+	void customerorder(int item1,int item2,int item3,int item4) {
+		if(maggie==0)
+			System.out.println("[!] Sorry Maggie is running out of stock."); 
+		else if (item1>maggie) 
+			System.out.println("Sorry quantity ordered for maggie cannot be fulfilled.");
+			else maggie = maggie-item1;
```
Comment (harshvegada) [inline src/prasad/Shop.java:14]: Here, You should display available stock for Maggie. with in else if() condition
remove else statement from line 14th

same for others

**Review by harshvegada — COMMENTED** [review]
Work on given comment, rest logic looks good to me!

File: src/prasad/Shop.java (hunk 1 above)
Comment (PrasadCOct2020) [reply src/prasad/Shop.java:14]: Hi @harshvegada I have included additional condition to check if customer has ordered more items than what are available in store and then convey the user with an error that stated quantity cannot be fulfilled. I have written separate method to display items available in inventory post accepting item counts from user. I have tested the code and it works fine.

**Review by harshvegada — APPROVED** [review]
got it!

**Pull request merged**